test/e2e/node_disconnect/lib/node_disconnect_lib.go

```diff
@@ -144,21 +162,60 @@ func (env *DisconnectEnv) LossWhenIdleTest() {
 	common.RunFio(env.fioPodName, 20)
 }
 
+// Run fio against the cluster while a replica mayastor pod is unscheduled and then rescheduled
+func (env *DisconnectEnv) PodLossTest() {
+	fmt.Printf("removing mayastor pod from node \"%s\"\n", env.nodeToIsolate)
+	SuppressMayastorPodOn(env.nodeToIsolate)
+
+	fmt.Printf("running fio against volume\n")
+	common.RunFio(env.fioPodName, 20)
+
+	fmt.Printf("waiting up to %s for pod removal to affect the nexus\n", disconnectionTimeoutSecs)
+	Eventually(func() string {
+		return common.GetMsvState(env.uuid)
+	},
+		disconnectionTimeoutSecs, // timeout
+		"1s",                     // polling interval
+	).Should(Equal("degraded"))
+
+	fmt.Printf("volume is in state \"%s\"\n", common.GetMsvState(env.uuid))
+
+	fmt.Printf("running fio against the degraded volume\n")
+	common.RunFio(env.fioPodName, 20)
+
+	fmt.Printf("enabling mayastor pod on node \"%s\"\n", env.nodeToIsolate)
+	env.UnsuppressMayastorPod()
+
+	fmt.Printf("running fio while volume is being repaired\n")
+	common.RunFio(env.fioPodName, 20)
```

Comment:
Is it possible that we could be running fio for a period where the volume is not being repaired. For example, maybe k8s is for some reason taking a long time to "unsuppress" the Mayastor pod. Because we only run fio for 20 seconds, maybe the unsuppress occurs after this. 

If would be good if we could run fio until the volume is successfully repaired, if possible.

Reply:
It just occurred to me I could run fio inside the Eventually() "loop".

Reply:
Fixed to run fio repeatedly until the msv state changes.